src/main/java/com/conveyal/datatools/manager/persistence/FeedStore.java

```diff
@@ -202,66 +214,45 @@ public File getFeed (String id) {
     /**
      * Create a new feed with the given ID.
```

Comment:
I feel like this method doc needs a little more info about what it does.

Reply:
I'm actually not sure what this does.  Help?

Reply:
This is obviously legacy code, so maybe it's outside the scope of your changes, but since you touched the method it made me notice that (to me) the javadoc doesn't really describe at all what the method does especially since the storeFeedLocally method has been removed from the internals. I would suggest that it say something like: 

```
Store GTFS file locally. This method is used when a new feed version or generated GTFS file
(e.g., the product of merging multiple GTFS files from a project) needs to be stored locally for
future use. Note: uploading the file to S3 is handled elsewhere as a finishing step, e.g., at the
conclusion of a successful feed processing/validation step.
```